### ISSUE TITLE
The website doesn't load any of my songs

### ISSUE THREAD

**Issue by Eugleo** [Shpigford/plylst#93]
Hey, even after ~3 hours the website doesn't seem to have loaded any of my songs. I've created a new playlist that "Uses only songs I've liked. Sorted by most popular. Limited to 100 songs." and I just get the "No songs match your filters." message.

**Comment by M1crogravity** [comment]
Same for me

**Comment by Shpigford** [comment]
Hey guys, sorry about the delay. Are you still having issues? Rolled out some performance fixes that 🤞hopefully fixes those issues.

**Comment by confidentjohn** [comment]
Hi
I just signed up today on the website app. I am having the same issue.

The stream section is blank.

If i create a playlist referencing Full Spotify Catalog, tracks get returned in a playlist.

However, for example i tried to make a playlist with this configuration

- Name - Loved Radiohead
- Auto Update - Yes
- Privacy - Public
- Source - Only Songs I have liked
- Rules, Artist Name  - Contains - Radiohead

This returns blank

I created some other playlists like, playcount is less than 2 for liked songs, which also returns empty (This is figure is correct behavior as i only just connected to the app, however as the stream isnt updating im not sure it will work)

Really look forward to using this app though. Thank you for creating.
